src/ck-ldb/HybridBaseLB.h

```diff
@@ -297,7 +297,8 @@ class HybridBaseLB : public CBase_HybridBaseLB
   ~HybridBaseLB();
 
   static void staticAtSync(void*);
-  void AtSync(void); // Everything is at the PE barrier
+  void AtSync(void){CmiAbort("\nAtSync not implemented");}; // Everything is at the PE barrier
```

Comment:
I assume this placeholder is going to be dealt with when we move to TreeLB?

Reply:
Not entirely. I've removed the abort from here because NAMD still can use an LB that extends HybridBaseLB, so this will stay around for now.